internal/featuretests/v3/envoy.go

```diff
@@ -251,6 +251,25 @@ func withSessionAffinity(route *envoy_route_v3.Route_Route) *envoy_route_v3.Rout
 	return route
 }
 
+type headerTerminalPair struct {
```

Comment:
Had to do this because a map doesn't iterate in the same order every time, would make tests flaky